paper-ieee-trans.tex

```diff
@@ -442,13 +436,13 @@ \subsubsection{SQA criteria}
 
 The SQA criteria are a set of conventions and recommendations that pave the way for
 the adequate development, timely delivery and reliable operation of the produced software components.
-It emphasizes the quality requirements and best practices to be applied at the
-development phase, such as style and testing compliance or human code reviews, to protect the
+They emphasize the quality requirements and best practices to be applied at the
+development phase, such as style and testing compliance or human malfunctions code reviews, to protect the
 production source code versions. Furthermore, the criteria promote the adoption of the software
 by providing minimum requirements of documentation content and stressing the usage of automated
 solutions for software deployment. The SQA criteria are continuously evolving \cite{sqa-baseline}, aiming at serving
 as a reference for future software development projects within the European research ecosystem.
-
+Some of the minimum recommended criteria are:
```

Comment:
-1 this is the full criteria, otherwise we will be asked for that

Reply:
+1 You can change as follows: The recommended criteria are: